tests/conftest.py

```diff
@@ -0,0 +1,61 @@
+# Fixtures for doctests
+from pathlib import Path
+import pytest
+from time import perf_counter
+from datetime import timedelta
+import os
+import sys
+from pytest_pretty import CustomTerminalReporter
+
+from _pytest.pathlib import bestrelpath
+
+
+@pytest.fixture(autouse=True)
+def handle_np2():
+    # TODO: remove when we require numpy >= 2
+    try:
+        import numpy as np
+
+        np.set_printoptions(legacy="1.21")
+    except ImportError:
+        pass
+
+
+class SKTerminalReporter(CustomTerminalReporter):
+    """
+    This ia s custom terminal reporter to how long it takes to finish given part of tests.
+    It prints time each time when test from different file is started.
+
+    It is created to be able to see if timeout is caused by long time execution, or it is just hanging.
```

Comment:
```suggestion
    Custom terminal reporter to display test runtimes.
    It displays the cumulative runtime each time a new file is tested.
```

I noticed a few typos in the above paragraph, so re-wrote it.
Not sure if "cumulative runtime" is the right term, but you get the general idea.